hbase-server/src/main/java/org/apache/hadoop/hbase/coprocessor/RegionCoprocessorEnvironment.java

```diff
@@ -130,4 +131,13 @@
    * @return the RawCellBuilder
    */
   RawCellBuilder getCellBuilder();
+
+  /**
+   * Provide server type to let users know the current context, it is on region server or compaction
+   * server or replication server, then user could choose different implementations.
+   * 1. through rpc call: {@link #getConnection()}
```

Comment:
The comment is a bit confusing if users do not know our previous discussion. Better describe like this:

"Provide server type to let users know the current context, it is on region server or compaction server or replication server, then users could choose different implementations. For example, on compaction server, you can not get the region instance, so if you want to get something from a region, maybe you should go with getConnection(), to communicate with the actual region instance through rpc. But if you are on region server, then you can just use getRegion or getOnlineRegions to get the region instance directly."

Reply:
fixed